lib/galaxy/jobs/runners/kubernetes.py

```diff
@@ -226,7 +204,21 @@ def __get_k8s_job_spec_template(self, ajs):
         (see pod selector) and an appropriate restart policy."""
         k8s_spec_template = {
             "metadata": {
-                "labels": {"app": self.__produce_unique_k8s_job_name(ajs.job_wrapper.get_id_tag())[:-5]}
+                "labels": {
+                    "app.kubernetes.io/name": ajs.job_wrapper.tool.tool_id,
+                    "app.kubernetes.io/instance": self.__produce_unique_k8s_job_name(ajs.job_wrapper.get_id_tag()),
+                    "app.kubernetes.io/version": ajs.job_wrapper.tool.version,
+                    "app.kubernetes.io/component": "tool",
```

Comment:
```suggestion
                    "app.kubernetes.io/component": "tool",
```
Should this be `job` ?

Reply:
Nice catch. Have pushed in the change.

Reply:
the component is a tool, the job is the reason for running it